app/slack_bot.js

```diff
@@ -259,6 +259,17 @@ controller.hears(['^invite.*\\|(.*)>'],
         });
   });
 
+controller.hears(['^allowed domains$', '^how to invite$', '^join this Slack$'], function(bot, message) {
```

Comment:
So this works, exactly as you expect, it will respond to any mention of "allowed domain" for example.

However the ^$ are anchors, so in this case, it would only respond to an entire message "allowed domain", so "what is the allowed domain" wouldn't match.